### PR TITLE
DEVDOCS-4913: [update] Webhooks, update store/customer/updated

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4913]

## What changed?
- Edit description for store/customer/updated

[DEVDOCS-4913]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by slsriehl — DISMISSED** [review]

**Review by isaacchen-bc — APPROVED** [review]

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 1]
```diff
@@ -281,7 +281,7 @@ Consult the [channel events section of the Channel Webhooks Guide](/api-docs/cha
 |:-------------------------|:------------|
 | store/customer/* | Fires for all store/customer events. |
 | store/customer/created | Fires when a new customer is created.|
-| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. |
+| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. Tracks changes to customer attributes only if the changes are made using the Control Panel. |
```
Comment (isaacchen-bc) [inline docs/api-docs/webhooks/webhook-events.mdx:284]: One thing I thought worth mentioning is that updating customer attributes in CP would only trigger the same event (same type and same payload) as updating the customer. So, the information such as what customer attributes have been changed to would **not** show in the event.

Comment (bc-andreadao) [reply docs/api-docs/webhooks/webhook-events.mdx]: 👍 Thanks, I updated the descriptions based on your feedback

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 2]
```diff
@@ -281,7 +281,7 @@ Consult the [channel events section of the Channel Webhooks Guide](/api-docs/cha
 |:-------------------------|:------------|
 | store/customer/* | Fires for all store/customer events. |
 | store/customer/created | Fires when a new customer is created.|
-| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. |
+| store/customer/updated | Fires when a customer is updated. Does not currently track changes to the customer address. <br /><br />Tracks changes to customer attributes only if the changes are made through the Control Panel. This triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes. |
```
Comment (bc-traciporter) [inline docs/api-docs/webhooks/webhook-events.mdx:284]:
```suggestion
| store/customer/updated | Fires when a customer is updated. It does not currently track changes to the customer address. <br /><br />Tracks changes to customer attributes only if you make the changes through the control panel. This change triggers the same event `type` and payload as updating a customer; the payload does not include customer attributes. |
```

**Review by bc-traciporter — COMMENTED** [review]
same comment as PR 1408

**Review by bc-traciporter — COMMENTED** [review]
same as PR 1408

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**